visual_behavior/data_access/loading.py

```diff
@@ -489,17 +489,24 @@ def get_ophys_dataset(ophys_experiment_id, include_invalid_rois=False, from_lims
     Arguments:
         ophys_experiment_id {int} -- 9 digit ophys experiment ID
         include_invalid_rois {Boolean} -- if True, return all ROIs including invalid. If False, filter out invalid ROIs
-        from_lims -- if True, loads dataset directly from BehaviorOphysSession.from_lims(). Invalid ROIs will be included.
-        from_nwb -- if True, loads dataset directly from BehaviorOphysSession.from_nwb_path(). Invalid ROIs will not be included.
+        load_from_lims -- if True, loads dataset directly from BehaviorOphysSession.from_lims(). Invalid ROIs will be included.
+        load_from_nwb -- if True, loads dataset directly from BehaviorOphysSession.from_nwb_path(). Invalid ROIs will not be included.
 
         If both from_lims and from_nwb are set to False, data will be loaded using the LIMS API then passed to the BehaviorOphysDataset class which allows invalid ROIs to be filtered out, and allows access to extended_stimulus_presentations, and face movie data.
 
     Returns:
         object -- BehaviorOphysSession or BehaviorOphysDataset instance, which inherits attributes & methods from SDK BehaviorOphysSession
     """
-    if from_lims:
+
+    id_type = from_lims.get_id_type(ophys_experiment_id)
+    if id_type != 'ophys_experiment_id':
+        warnings.warn('It looks like you passed an id of type {} instead of an ophys_experiment_id'.format(id_type))
+
+    assert id_type == 'ophys_experiment_id', "The passed ID type is {}. It must be an ophys_experiment_id".format(id_type)
+
+    if load_from_lims:
         dataset = BehaviorOphysExperiment.from_lims(int(ophys_experiment_id))
-    elif from_nwb:
+    elif load_from_nwb:
```

Comment:
wondering if it's worth adding a descriptive error message here if the experiment is not found in lims or nwb; just a suggestion.

Reply:
There is an existing statement that will print a line if there is no NWB file found:
https://github.com/AllenInstitute/visual_behavior_analysis/blob/7f04506065fc9767b41b248cdfa300f9409786bf/visual_behavior/data_access/loading.py#L511

The changes in PR #760 will cause an assertion error to be thrown if there is no associated experiment ID in LIMS.

Are there any other specific checks you're thinking of?